package.json

```diff
@@ -32,7 +33,7 @@
     "install:node-v8": "ts-node ./scripts/install-node-v8.ts",
     "build": "tsc -p ./src && tsc lib/tc.ts",
     "watch-build": "tsc -w -p ./src && tsc -w -p ./lib",
-    "test": "./node-v8/bin/node ./node_modules/jest/bin/jest.js ",
+    "test": "bash scripts/test",
```

Comment:
Yarn test does not work on all platforms. Nor does npm test. It's fine for Linux/OSX but not Windows